### ISSUE TITLE
ExternalName Service no Pods

### ISSUE THREAD

**Issue by protometa** [pulumi/pulumi-kubernetes#189]
Service of type ExternalName tries to wait for pods and errors on create:

```
Diagnostics:
  kubernetes:core:Service: gummicube-site
    error: Plan apply failed: 2 errors occurred:

    * Timeout occurred for 'gummicube-site'
    * Service does not target any Pods
```

Though the service is correct created. Warnings on subsequent update:

```
Diagnostics:
  kubernetes:core:Service: gummicube-site
    warning: This resource failed to initialize in a previous deployment. It is recommended
             to update it to fix these issues:
               - Problem #1 Timeout occurred for 'gummicube-site'
               - Problem #2 Service does not target any Pods
```

**Comment by hausdorff** [comment]
@protometa actually, we just solved this in #180. _cf_., #178.

**Comment by hausdorff** [comment]
BTW, the release containing this change will happen imminently -- tomorrow, probably.
